src/RoslynInsertionTool/RoslynInsertionTool.Commandline/RoslynInsertionToolCommandline.cs

```diff
@@ -151,6 +151,11 @@ partial class RoslynInsertionToolCommandline
                 $"Defaults to \"{options.InsertWillowPackages}\".",
                 insertWillowPackages => options = options.WithInsertWillowPackages(bool.Parse(insertWillowPackages))
             },
+            {
+                "il=|insertLibraryPackages=",
```

Comment:
Can we redo these switches to something a bit clearer? Perhaps "update packages in CoreXT" and "add binding redirect information" to more clearly define the behavior of these two?

Reply:
hmm, looks like there is a `UpdateCoreXTLibraries` switch that we could potentially reuse. Dont know if that has other implications to it. [Here](https://github.com/dotnet/roslyn-tools/blob/master/src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs#L101) is where it is used today and its default is false. Is this set to true for roslyn and project system insertions?